cms/static/js/views/course_info_update.js

```diff
@@ -70,31 +97,35 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
             },
 
             handleValidationError: function(model, error) {
-                var ele = this.$el.find('#course-update-list li[name=\"' + model.cid + '\"]');
-                $(ele).find('.message-error').remove();
+                var validationElement = this.$el.find('#course-update-list li[name="' + model.cid + '"]');
+                $(validationElement).find('.message-error').remove();
                 for (var field in error) {
                     if (error.hasOwnProperty(field)) {
-                        $(ele).find('#update-date-' + model.cid).parent().append(
-                        this.errorTemplate({message: error[field]})
-                    );
-                        $(ele).find('.date-display').parent().append(this.errorTemplate({message: error[field]}));
+                        HtmlUtils.append(
+                            $(validationElement).find('#update-date-' + model.cid).parent(),
+                            this.errorTemplate({message: error[field]})
+                        );
+                        HtmlUtils.append(
+                            $(validationElement).find('.date-display').parent(),
+                            this.errorTemplate({message: error[field]})
+                        );
                     }
                 }
-                $(ele).find('.save-button').addClass('is-disabled');
+                $(validationElement).find('.save-button').addClass('is-disabled');
             },
 
             validateModel: function(model) {
+                var validationElement = this.$el.find('#course-update-list li[name="' + model.cid + '"]');
                 if (model.isValid()) {
-                    var ele = this.$el.find('#course-update-list li[name=\"' + model.cid + '\"]');
-                    $(ele).find('.message-error').remove();
-                    $(ele).find('.save-button').removeClass('is-disabled');
+                    $(validationElement).find('.message-error').remove();
+                    $(validationElement).find('.save-button').removeClass('is-disabled');
```

Comment:
I think we will have to revisit this code in the future to make the save button _actually_ disabled (instead of just CSS class). But out of scope for this change.